docs/general/run-a-node/set-up-your-node/run-a-paratime-node.mdx

```diff
@@ -92,10 +91,9 @@ To register a node that is both a validator and a ParaTime node, the entity for
 
 * Entity registration staking threshold (currently 100 tokens),
 * Validator node staking threshold (currently 100 tokens),
-* Compute node staking threshold (currently 100 tokens),
-* Storage node staking threshold (currently 100 tokens).
+* Compute node staking threshold (currently 50,000 tokens),
```

Comment:
This is incorrect, the previous value was correct. 50k is for registering a runtime.